### DISCUSSION TITLE
search plugin not refreshing

### DISCUSSION THREAD

**Discussion by ediphy-dcottrell** [docsifyjs/docsify#1504] [Q&A] [answered]
I've noticed the search plugin is now showing me stale results. I have reduced maxAge : 1000 * 30 which should 30 seconds but no luck so far. No errors show in dev tools. Any suggestions on debugging this further?

**Comment by sy-records** [comment, marked as answer]
I tested it locally with `docsify-cli` with no problems, if you use services like GitHub pages, you need to consider caching issues
